### PR TITLE
Reset member invite on success

### PR DESCRIPTION
**Pull request by tfussell**
Fixes https://github.com/venturemark/webclient/issues/327

### CODE REVIEW

File: src/component/addeditmembers/index.tsx [hunk 1]
```diff
@@ -31,7 +31,10 @@ interface AddEditMembersProps extends DefaultAddEditMembersProps {
 
 function AddEditMembers(props: AddEditMembersProps) {
   const { currentVenture, currentTimeline, user, ...rest } = props;
-  const { handleSubmit, control, reset, errors } = useForm({
+  const { handleSubmit, control, reset, errors } = useForm<{ email: string }>({
+    defaultValues: {
```
Comment (tfussell) [inline src/component/addeditmembers/index.tsx:35]: `reset()` doesn't work unless `defaultValues` is defined.

File: src/component/addeditmembers/index.tsx [hunk 2]
```diff
@@ -91,17 +94,23 @@ function AddEditMembers(props: AddEditMembersProps) {
...
+        fromName: user?.name ?? "",
+        resource: !currentTimeline ? "venture" : "timeline",
+        role: "member",
+        fromVentureName: currentVenture?.name ?? "",
+        email: data.email,
+        token,
+      },
+      {
+        onSuccess() {
+          reset();
```
Comment (tfussell) [inline src/component/addeditmembers/index.tsx:110]: Only reset on success.

File: src/module/hook/user/index.ts [hunk 3]
```diff
@@ -32,10 +32,6 @@ const getCurrentUser = async (searchCurrentUser: ISearchCurrentUser) => {
     const data = await api.API.User.Search(searchCurrentUser);
     return data;
   } catch (err) {
-    if (err.message === "index must not be empty") {
```
Comment (tfussell) [inline src/module/hook/user/index.ts:35]: No longer needed after https://github.com/venturemark/apiserver/pull/219

File: tsconfig.json [hunk 4]
```diff
@@ -6,8 +6,12 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "isolatedModules": true,
-    "jsx": "react",
-    "lib": ["dom", "dom.iterable", "esnext"],
+    "jsx": "react-jsx",
```
Comment (tfussell) [inline tsconfig.json:9]: Unsure how this got changed. This is required so that `React` is imported correctly.

**Review by marcusellison — APPROVED** [review]

**Pull request merged**